mon-pix/app/routes/assessments/checkpoint.js

```diff
@@ -14,9 +18,17 @@ export default class CheckpointRoute extends Route {
     }
 
     if (assessment.isSmartPlacement) {
-      const campaigns = await this.store.query('campaign', { filter: { code: assessment.codeCampaign } });
+      let campaign = null;
+      const code = assessment.codeCampaign;
+      const campaignsInStore = this.peeker.find('campaign', (campaign) => campaign.code === code);
```

Comment:
Si on avait l'id de la campagne dans l'objet assessment. On aurait pu utilisé nativement le peekRecord qui lui aussi ne fait pas d'appel réseau. 

const campaignsInStore = this.store.peekRecord('campaign', campaignId);